src/core/util/getAbsoluteUrl.js

```diff
@@ -6,6 +6,12 @@
  * @returns {String} Full URL
  */
 export default function getAbsoluteUrl(rootUrl, path = "") {
+  // Check if the path is already absolute
+  const pathStart = path.slice(0, 7).toLowerCase();
+  if (pathStart.startsWith("https://") || pathStart.startsWith("http") || pathStart.startsWith("//")) {
```

Comment:
This should be `.slice(0, 8)`. Otherwise `https://` becomes `https:/` and will never pass the first check. I think the only reason your test is passing is because of `pathStart.startsWith("http")`, which should be `pathStart.startsWith("http://")` to prevent the `http.jpeg` case that @focusaurus mentioned.

I suggest adding a test with `http.jpeg` relative URL and make sure it is properly converted to absolute.